.github/workflows/live-tests.yml

```diff
@@ -0,0 +1,198 @@
+
+      - name: Cleanup
+        if: always()
+        run: docker rm -f ${{ env.CONTAINER_NAME }} || true
+
+  create-pr:
+    needs: live-tests
+    if: always() && needs.live-tests.outcome == 'success' && github.event_name != 'pull_request'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+
+      - name: Extract all recordings and create PR
+        env:
+          LLAMA_STACK_REPO: https://github.com/llamastack/llama-stack.git
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          REPO_RECORDINGS_DIR="tests/integration/recordings"
+          [ ! -d "$REPO_RECORDINGS_DIR" ] && exit 0
+          
+          # Check if there are any recordings
+          [ -z "$(find "$REPO_RECORDINGS_DIR" -type f)" ] && exit 0
+          
+          # Clone llama-stack and update recordings
+          LLAMA_STACK_CLONE_DIR=$(mktemp -d)
+          git clone "https://x-access-token:$GITHUB_TOKEN@github.com/llamastack/llama-stack.git" "$LLAMA_STACK_CLONE_DIR"
+          cd "$LLAMA_STACK_CLONE_DIR"
+          
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+          
+          BRANCH_NAME="update-recordings-$(date +%Y%m%d-%H%M%S)-${{ github.run_id }}"
+          git checkout -b "$BRANCH_NAME"
+          
+          TARGET_RECORDINGS_DIR="$LLAMA_STACK_CLONE_DIR/tests/integration/recordings"
+          mkdir -p "$TARGET_RECORDINGS_DIR"
+          
+          # Copy all recordings
+          find "$REPO_RECORDINGS_DIR" -type f | while IFS= read -r recording; do
+            relative_path=$(echo "$recording" | sed "s|^$REPO_RECORDINGS_DIR/||")
+            mkdir -p "$(dirname "$TARGET_RECORDINGS_DIR/$relative_path")"
+            cp "$recording" "$TARGET_RECORDINGS_DIR/$relative_path"
+          done
+          
+          git diff --quiet && exit 0
+          
+          git add tests/integration/recordings/
+          git commit -m "chore: update test recordings from live tests
+
+          Updated recordings from successful live CI tests run on $(date -u +%Y-%m-%d).
+          Generated by: ${{ github.workflow }} workflow
+          Run: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+          
+          git push origin "$BRANCH_NAME" || exit 1
+          
+          gh auth setup-git
+          gh pr create \
+            --repo llamastack/llama-stack \
+            --title "chore: update test recordings" \
+            --body "This PR updates test recordings from successful live CI tests.
+
+          **Source**: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+          **Date**: $(date -u +%Y-%m-%d)
+
+          Auto-generated by the live CI tests workflow." \
```

Comment:
We're going to have a lot of commits with this message. You can delete this `Auto-generated` line - it's just repetitive noise since we're recording `Source` and `Date`.